docs/user_guide/hopsworks/jupyter.rst

```diff
@@ -0,0 +1,227 @@
+    :alt: Starting a spark job from Jupyter
+    :target: `jupyter6.png`_
+    :align: center
+    :figclass: align-center
+
+    Starting a spark job from Jupyter
+
+
+When you execute a cell in Jupyter that starts a Spark job, you can go back to the Hopsworks-Jupyter-UI and you will se that a link to the SparkUI for the job have been created.
+
+.. _jupyter11.png: ../../_images/jupyter11.png
+.. figure:: ../../imgs/jupyter11.png
+    :alt: Opening the SparkUI in Hopsworks
+    :target: `jupyter11.png`_
+    :align: center
+    :figclass: align-center
+
+    Opening the SparkUI in Hopsworks
+
+
+.. _jupyter12.png: ../../_images/jupyter12.png
+.. figure:: ../../imgs/jupyter12.png
+    :alt: The SparkUI in Hopsworks
+    :target: `jupyter12.png`_
+    :align: center
+    :figclass: align-center
+
+    The SparkUI in Hopsworks
+
+In addition to having access to a regular python interpreter as well as the spark cluster, you also have access to **magic** commands provided by sparkmagic. You can view a list of all commands by executing a cell with `%%help`:
+
+.. _jupyter7.png: ../../_images/jupyter7.png
+.. figure:: ../../imgs/jupyter7.png
+    :alt: Printing a list of all sparkmagic commands
+    :target: `jupyter7.png`_
+    :align: center
+    :figclass: align-center
+
+    Printing a list of all sparkmagic commands
+
+Plotting with Pyspark Kernel
+-------------------
+So far throughout this tutorial, the Jupyter notebook have behaved more or less identical to how it does if you start the notebook server locally on your machine using a python kernel, without access to a Hadoop cluster. However, there is one main difference from a user-standpoint when using pyspark notebooks instead of regular python notebooks, this is related to *plotting*.
+
+Since the code in a pyspark notebook is being executed remotely, in the spark cluster, regular python plotting will not work. What you can do however, is to use sparkmagic to download your remote spark dataframe as a local pandas dataframe and plot it using matplotlib, seaborn, or sparkmagics built in visualization. To do this we use the magics: `%%sql`, `%%spark`, and `%%local`. The steps to do plotting using a pyspark notebook are illustrated below. Using this approach, you can have large scale cluster computation and plotting in the same notebook.
+
+**Step 1 : Create a remote Spark Dataframe**:
+
+.. _jupyter8.png: ../../_images/jupyter8.png
+.. figure:: ../../imgs/jupyter8.png
+    :alt: Creating a spark dataframe
+    :target: `jupyter8.png`_
+    :align: center
+    :figclass: align-center
+
+    Creating a spark dataframe
+
+**Step 2 : Download the Spark Dataframe to a local Pandas Dataframe using %%sql or %%spark**:
+
+Note: you should **not** try to download large spark dataframes for plotting. When you plot a dataframe, the entire dataframe must fit into memory, so add the flag `--maxrows x` to limit the dataframe size when you download it to the local Jupyter server for plotting.
```

Comment:
@Limmen We can stress out this point even more by bolding the `Note` as well or something like that

Reply:
Agree, fixed